Sources/Testing/Issues/Issue+Recording.swift

```diff
@@ -82,10 +83,12 @@ extension Issue {
   /// Use this function if, while running a test, an issue occurs that cannot be
   /// represented as an expectation (using the ``expect(_:_:sourceLocation:)``
   /// or ``require(_:_:sourceLocation:)-5l63q`` macros.)
-  @_spi(Experimental)
+  /// @Metadata {
```

Comment:
For each of these `@Metadata` blocks, we typically include a blank line above them to separate them from ordinary documentation